amqp/src/main/scala/org/apache/pekko/stream/connectors/amqp/AmqpConnectorSettings.scala

```diff
@@ -28,6 +28,7 @@ import scala.concurrent.duration._
 sealed trait AmqpConnectorSettings {
   def connectionProvider: AmqpConnectionProvider
   def declarations: immutable.Seq[Declaration]
+  def avoidArrayCopy: Boolean
```

Comment:
Maybe just copyArray?

Reply:
With configs, I prefer the default to be false. If I change this to be copyArray, I'll have to make the default true - because is the current 1.0.0 behaviour.

Reply:
You can also use something like `reuseByteArray` (if you want to use a single verb instead of `avoid[Verb]`)

Reply:
thanks @JD557, I changed to use the name you suggested